src/cli/commands/check.py

```
@@ -84,58 +84,80 @@ def check(
-        _run_type_ignore_analysis(target_path, verbose)
+        elif focus == "ignore":
+            _run_type_ignore_analysis(target_path, verbose=verbose, exclude_patterns=exclude_patterns)
 
-    elif focus == "quality":
-        _run_quality_check(target_path, config, verbose)
+        elif focus == "quality":
+            _run_quality_check(target_path, config, verbose=verbose, exclude_patterns=exclude_patterns)
 
 
-def _run_type_analysis(target_path: Path, verbose: bool) -> None:
+def _run_type_analysis(target_path: Path, verbose: bool, exclude_patterns: list[str] | None = None) -> None:
```

Ruff (0.14.0)

Boolean-typed positional argument in function definition

(FBT001)